cmake-proxies/cmake-modules/CopyLibs.cmake

```diff
@@ -94,7 +101,7 @@ function( gather_libs src )
 
          set(line "${WXWIN}/${line}")
```

Comment:
Clearer would be 
set(lib "${WXWIN}/${line}")
and then adjust line 104-107 to use lib instead of line.  Makes it easier to check that the recursion guard is guarding the right thing.